From 76f7e61f1a7b3e3d445b4f6c32e29c5f05fa2c71 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Tue, 30 May 2023 20:27:03 +0000
Subject: [PATCH] IWYU: add cmath for various math functions in //skia

Bug: 957519
Change-Id: I4616f7f161330feb3fb2299da094874663655189
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4545069
Commit-Queue: Nico Weber <thakis@chromium.org>
Reviewed-by: Nico Weber <thakis@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1150817}
---
 skia/ext/skcolorspace_trfn.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/skia/ext/skcolorspace_trfn.cc b/skia/ext/skcolorspace_trfn.cc
index c0d6bd2a80b2d..97c4dfad97438 100644
--- a/skia/ext/skcolorspace_trfn.cc
+++ b/skia/ext/skcolorspace_trfn.cc
@@ -4,6 +4,8 @@
 
 #include "skia/ext/skcolorspace_trfn.h"
 
+#include <cmath>
+
 namespace skia {
 
 namespace {
-- 
2.41.0

